.travis.yml

```diff
@@ -10,12 +10,14 @@ install:
   # https://github.com/travis-ci/travis-ci/issues/8408
   - unset _JAVA_OPTIONS
   # download Cloud SDK
-  - wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-200.0.0-linux-x86_64.tar.gz
-  - tar -xzf google-cloud-sdk-200.0.0-linux-x86_64.tar.gz -C /home/travis
+  - wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-227.0.0-linux-x86_64.tar.gz
```

Comment:
Could we just use `https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.tar.gz` to get the latest?  After all, it's what the Managed SDK's `FileResourceProviderFactory` does…

Reply:
That runs the risk of breaking our build through no change of our own. That might or might not be a good thing.

Reply:
Yes. However it may not be immediately obvious that the failure is due to that and not due to change sin the PR itself.

Reply:
We run "update" anyway, so our tests are acting on the latest SDK.

The update I added does a little more caching and should speed up builds a little bit.